openreview/conference/matching.py

```diff
@@ -251,6 +251,92 @@ def _build_conflicts(self, submissions, user_profiles, get_profile_info):
             return self._build_profile_conflicts(ac_profiles, user_profiles)
         return self._build_note_conflicts(submissions, user_profiles, get_profile_info)
 
+    def append_note_conflicts(self, user_profile, build_conflicts=None):
```

Comment:
Can you call it `profile_id` same as `set_matching_conflicts`?

```suggestion
    def append_note_conflicts(self, profile_id, build_conflicts=None):
```